src/redux/books/books.js

```diff
@@ -0,0 +1,43 @@
+const BOOK_ADDED = 'BOOK_ADDED';
+const BOOK_REMOVED = 'BOOK_REMOVED';
+
+let lastId = 0;
+
+export const bookAddedAction = (book) => ({
+  type: BOOK_ADDED,
+  payload: {
+    title: book.title,
+    author: book.author,
+    category: book.category,
+  },
+});
+
+export const bookRemovedAction = (id) => ({
+  type: BOOK_ADDED,
+  payload: {
+    id,
+  },
+});
```

Comment:
- [ ] I believe, here, you wanted to say `BOOK_REMOVED`? 🤔

Reply:
Sorry about that mistake. I will change it.